framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -85,6 +106,10 @@ XYDelaunayGenerator::validParams()
 
   params.addClassDescription("Triangulates meshes within boundaries defined by input meshes.");
 
+  params.addParamNamesToGroup(
+      "use_auto_area_func auto_area_func_default_size auto_area_func_default_size_dist",
+      "Automatic Area Function");
```

Comment:
```suggestion
      "Automatic triangle meshing area control");
```
I would add the other parameters for that region in another parameter group